### ISSUE TITLE
Disable creation of new terms using the autocomplete widget

### ISSUE THREAD

**Issue by milossramek** [enu-sav/RS#7]
In entering a taxonomy term using the autocomplete widget one can enter a nonexisting term (when none of the offered items is selected). This possibility should be disabled, so that only predefined terms are entered.  A solution seems to be described here: https://www.drupal.org/node/1078878

Priority: low

**Comment by miromichalicka** [comment]
I started working on this feature in new branch https://github.com/enu-sav/RS/tree/feature/%237. I need to know to which fields we need to integrate this new functionality.

**Comment by milossramek** [comment]
it is related to fields, where taxonomy values Lexikalna skupina are added in heslo, importovane heslo and ilustracia.

**Comment by milossramek** [comment]
I've installed the Taxonomy Autocomplete Permission module, https://www.drupal.org/project/taxonomy_autocomplete_permission, it seems to work well: # a2a0b3b 
Its settings are not recognized by the features module, do we need some extra settings for that?

**Comment by miromichalicka** [comment]
I you want to disable it completely, there is no need for any special module. If you want to enable it only for specific roles, ok. That module basically provides additional permissions, so you will find its settings there

**Comment by milossramek** [comment]
Yes, I want to disable creation of new items by autocomplete for all users, However, I did not manage to find such option. Where is it?

**Comment by miromichalicka** [comment]
Follow changes in 57979b4a9f98a69e336bfbbfa68a67f3e46f5da7
